File: src/main/java/client/Sounds.java
package client;

import java.util.HashMap;

public class Sounds {
    private static final HashMap<String, Sound> sounds  = new HashMap<>();
    public static Sound rain = new Sound("raindrop");
    public static Sound block_break = new Sound("block_break");

    public static void init(){
        register(rain);
        register(block_break);
    }

    private static void register(Sound sound){
        sounds.put(sound.getName(), sound);
    }

    public static void playSound(String name){
        sounds.get(name).play();
    }
}

File: src/main/kotlin/common/sounds/Sounds.kt
package common.sounds

import common.Game
import java.util.HashMap

object Sounds {
    private val sounds = HashMap<String, Sound>()
    var rain = Sound("raindrop", Game.DOMAIN)
    var block_break = Sound("block_break", Game.DOMAIN)

    fun init() {
        register(rain)
        register(block_break)
    }

    private fun register(sound: Sound) {
        sounds[sound.name] = sound
    }

    fun playSound(name: String) {
        sounds[name]?.play()
    }
}
